packages/frontend/src/components/DashboardTiles/TileForms/TileUpdateModal.tsx

```diff
@@ -63,15 +66,27 @@ const TileUpdateModal = <T extends Tile>({
     });
 
     return (
-        <Dialog
-            lazy
-            title="Edit tile content"
+        <Modal
+            size="xl"
+            title={
+                <Flex gap="xs" align="center">
```

Comment:
Group spacing='xs' is the same right? just wondering no need to change

Reply:
it really is... Thanks for spotting this. I will actually change and merge!